aries-site/src/themes/aries.js

```diff
@@ -19,7 +19,8 @@ const fontSizing = factor => {
   };
 };
 
-export const aries = deepMerge(hpe, {
+// export const aries = deepMerge(hpe, {
```

Comment:
Anchor loses color (need to set anchor color = 'text' so that it is white): 
![Screen Shot 2020-02-03 at 3 49 00 PM](https://user-images.githubusercontent.com/1756948/73698009-39197180-469d-11ea-8121-d2e675d907b9.png)

Reply:
![Screen Shot 2020-02-03 at 3 55 56 PM](https://user-images.githubusercontent.com/1756948/73698216-ba710400-469d-11ea-9b7c-66acec5d4bf2.png)

Reply:
Good catch on that